### PR TITLE
feat!: Changed type to _type

### PR DESCRIPTION
**Pull request by taorepoara**
## About this PR

Adapt to API breaking changes:

- [x] `type` => `_type`
- [x] `action` => `listener`
- [x] remove deprecated elements

## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I did not include breaking changes
- [x] I made my own code-review before requesting one

### I included unit tests that cover my changes
- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### I added/updated the documentation about my changes
- [ ] 📜 README.md
- [ ] 📕 docs/*.md
- [ ] 📓 docs.lenra.io
- [x] 🙅 no documentation needed

### CODE REVIEW

File: examples/showcase/web/manifest.json [hunk 1]
```diff
@@ -12,12 +12,12 @@
         {
             "src": "icons/Icon-192.png",
             "sizes": "192x192",
-            "type": "image/png"
+            "_type": "image/png"
```
Comment (jonas-martinez) [inline examples/showcase/web/manifest.json]: I think that this file shouldn't be impacted by the change. You should revert the changes here.

**Review by jonas-martinez — CHANGES_REQUESTED** [review]

File: examples/showcase/web/manifest.json (hunk 1 above)
Comment (taorepoara) [reply examples/showcase/web/manifest.json]: Fixed

**Review by jonas-martinez — APPROVED** [review]
